### ISSUE TITLE
An error occurred during spconv installation

### ISSUE THREAD

**Issue by CatherineZhou53** [hustvl/HAIS#35]
Hi, thanks for your excellent work. I tried to reproduce the result of the pretrained model on the test set of ScanNet v2.0. I followed the instructions in your repo to process the data and installation, but I faced this issue:
I am currently experiencing compilation errors with spconv 1.0 in Python 3.7, GPU version Pytorch 1.1, CUDA 10.1, gcc-5.4. In Python 3.7, GPU version Pytorch 1.1, CUDA 10.1, and gcc-5.4 environments, spconv2.3.3 was successfully installed by "pip install spconv", and an error occurred during HAIS model testing. The error was inferred to be caused by version mismatch. Spconv 1.2.1 was successfully installed in Python 3.7, CPU version Pytorch 1.8.1, CUDA 10.1, and gcc-7.5[1]. An error occurred when python import spconv[2]. What are the specific environmental conditions under which you can successfully run this algorithm?
![1](https://user-images.githubusercontent.com/55700912/222668774-768741ff-8026-43bf-8dca-07129578ca88.png)
![2](https://user-images.githubusercontent.com/55700912/222669151-84b80709-e882-4465-909b-7b2f8327f4b6.png)
